src/main/java/com/google/codeu/servlets/LogoutServlet.java

```diff
@@ -33,7 +33,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
wondering if this IOException should be caught. Would be a better user experience if they didn't have to deal with the case where the exception gets thrown and the page breaks

Reply:
the above comment goes for all of the exceptions being thrown across all the files.